.travis.yml

```diff
@@ -17,53 +18,36 @@ env:
     - ES_JAVA_OPTS="-Xms256m -Xmx512m"
 
   matrix:
-    - MODULE='berkeleyje'
-    - MODULE='cassandra'
-    - MODULE='es' ARGS='-DthreadCount=1'
-    - MODULE='es' ARGS='-DthreadCount=1 -Pelasticsearch2'
     - MODULE='hadoop-parent/janusgraph-hadoop-2'
-    - MODULE='hbase-parent/janusgraph-hbase-098'
-    - MODULE='hbase-parent/janusgraph-hbase-10'
-    - MODULE='lucene' ARGS='-DthreadCount=1'
-    - MODULE='solr' ARGS='-DthreadCount=1'
+    - MODULE='lucene'
```

Comment:
I found dependencies are often not initially cached so I wanted to put the quicker dependencies at the top to absorb possible dependency download time. It also seems to help with build parallelism to keep builds with similar time together.